lib/workers/repository/update/branch/auto-replace.ts

```diff
@@ -147,9 +152,22 @@ export async function doAutoReplace(
   if (reuseExistingBranch) {
     return await checkExistingBranch(upgrade, existingContent);
   }
+  const replaceWithoutReplaceString = Boolean(
+    is.string(newName) &&
+      newName !== depName &&
+      (is.undefined(upgrade.replaceString) ||
+        !upgrade.replaceString?.includes(depName!))
+  );
```

Comment:
```suggestion
  const replaceWithoutReplaceString = (
    is.string(newName) &&
      newName !== depName &&
      (is.undefined(upgrade.replaceString) ||
        !upgrade.replaceString?.includes(depName!))
  );
```

remove explicit `Boolean` function call